src/test/java/mil/dds/anet/test/resources/PersonResourceTest.java

```diff
@@ -52,6 +56,12 @@
   // 200 x 200 avatar
   final File DEFAULT_AVATAR =
       new File(PersonResourceTest.class.getResource("/assets/default_avatar.png").getFile());
+  private static PositionHistoryDao positionHistoryDao;
+
+  @BeforeAll
+  public static void setUpClass() {
+    positionHistoryDao = new PositionHistoryDao();
+  }
```

Comment:
Why not just `private static final PositionHistoryDao positionHistoryDao = new PositionHistoryDao();` ?